dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/OpenAireExporter.java

```diff
@@ -1,26 +1,32 @@
 package edu.harvard.iq.dataverse.export;
 
-import java.io.OutputStream;
-
-import javax.json.JsonObject;
-import javax.xml.stream.XMLStreamException;
-
-import com.google.auto.service.AutoService;
-
 import edu.harvard.iq.dataverse.DatasetVersion;
 import edu.harvard.iq.dataverse.export.openaire.OpenAireExportUtil;
 import edu.harvard.iq.dataverse.export.spi.Exporter;
 import edu.harvard.iq.dataverse.util.BundleUtil;
+import edu.harvard.iq.dataverse.util.json.JsonPrinter;
+
+import javax.json.JsonObject;
+import javax.xml.stream.XMLStreamException;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 
-@AutoService(Exporter.class)
 public class OpenAireExporter implements Exporter {
 
-    public OpenAireExporter() {
+    private boolean excludeEmailFromExport;
+
+    // -------------------- CONSTRUCTORS --------------------
+
+    public OpenAireExporter(boolean excludeEmailFromExport) {
+        this.excludeEmailFromExport = excludeEmailFromExport;
     }
 
+    // -------------------- LOGIC --------------------
+
     @Override
     public String getProviderName() {
-        return "oai_datacite";
+        return "openAire";
```

Comment:
It seems like this string has to match the value in ExporterConstant. So it seems smart move to use something like `ExporterConstant.OPENAIRE.toString()` or `ExporterConstant.OPENAIRE.getName()` that will return this string